compile.go

```diff
@@ -139,85 +139,6 @@ func isPreludePackage(pkg string) bool {
 	return false
 }
 
-type scopeSet struct {
```

Comment:
I think this was intended to implement `runtime.Prelude()`, but that is now done in a different way.